### PR TITLE
[PLT-6744] Add "Only visible to you" note for ephemeral messages

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Add "Only visible to you" note for ephemeral messages

#### Ticket Link
JIRA: https://mattermost.atlassian.net/browse/PLT-6744

#### Checklist
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates

### CODE REVIEW

**Comment by cpanato** [comment]
@esethna sure I will make the same size, I made a bit smaller because I think this is not the main information for the user.

**Comment by cpanato** [comment]
@esethna done! :)

**Comment by esethna** [comment]
Thanks @cpanato, [posted for UX approval](https://pre-release.mattermost.com/core/pl/wyn71wofd7dwfprzmfycim9pic) but looks good in testing!

File: webapp/components/post_view/post_info/post_info.jsx [hunk 1]
```diff
@@ -55,6 +55,11 @@ export default class PostInfo extends React.PureComponent {
         lastPostCount: PropTypes.number,
 
         /**
+         * Identify is the Display is using compact view
```
Comment (jwilander) [inline webapp/components/post_view/post_info/post_info.jsx]: Can we update this comment to `Set to render in compact view`

Comment (cpanato) [inline webapp/components/post_view/post_info/post_info.jsx]: done

**Pull request merged**